prov/gni/src/gnix_nic.c

```diff
@@ -50,7 +50,7 @@
 #define GNIX_VC_FL_MIN_SIZE 128
 
 static int gnix_nics_per_ptag[GNI_PTAG_USER_END];
-static DLIST_HEAD(gnix_nic_list);
+DLIST_HEAD(gnix_nic_list);
 static pthread_mutex_t gnix_nic_list_lock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
I think we need to open a different issue w.r.t. this line. We should be using our fastlock implementations.